v2/googlecloud-to-neo4j/src/main/java/com/google/cloud/teleport/v2/neo4j/options/Neo4jFlexTemplateOptions.java

```diff
@@ -39,8 +39,8 @@ public interface Neo4jFlexTemplateOptions extends CommonTemplateOptions {
   @TemplateParameter.GcsReadFile(
       order = 2,
       optional = true,
-      description = "Path to the Neo4j connection metadata",
-      helpText = "The path to the Neo4j connection metadata JSON file.")
+      description = "Path to the Neo4j connection JSON file, hosted on GCS.",
```

Comment:
I can also link to the schema if that helps (here and for the next option)
https://neo4j.com/docs/common-content-dataflow/0.1/_attachments/schemas/connection.v1.0.json